### PR TITLE
Fix microservices1

### PR DESCRIPTION
**Pull request by bojmil-armedia**

### CODE REVIEW

File: vagrant/provisioning/roles/arkcase-ldap-authority/tasks/main.yml [hunk 1]
```diff
@@ -4,4 +4,4 @@
...
+      java_opts: "-Dserver.port=9090 -Dspring.cloud.config.uri=https://{{ config_server_host | default(internal_host) }}/config -Dspring.profiles.include=server,runtime -Djava.net.preferIPv4Stack=true -Djavax.net.ssl.trustStorePassword={{ java_trust_store_pass }} -Djavax.net.ssl.keyStore={{ java_key_store }} -Djavax.net.ssl.trustStore={{ java_trust_store}} -Djavax.net.ssl.keyStorePassword={{ java_key_store_pass }} -Darkcase.ldap.url={{ ldap_portal_url | default(ldap_url) }} -Darkcase.ldap.base={{ ldap_portal_base | default(ldap_base) }} -Darkcase.ldap.manager-dn={{ ldap_portal_bind_user | default(ldap_bind_user) }} -Darkcase.ldap.manager-password={{ ldap_portal_bind_password | default(ldap_bind_password) }} -Darkcase.ldap.search-properties.user-domain={{ ldap_portal_user_domain | default(ldap_user_domain) }} -Darkcase.ldap.search-properties.user-search-base={{ ldap_portal_user_no_base | default(ldap_user_no_base) }} -Darkcase.ldap.search-properties.group-search-base={{ ldap_portal_group_no_base | default(ldap_group_no_base) }} -Darkcase.ldap.search-properties.user-prefix=portal."
```
Comment (bojmil-armedia) [inline vagrant/provisioning/roles/arkcase-ldap-authority/tasks/main.yml:7]: Right now we use this microservice only for the portal. We will need to change this when we start using it for ArkCase.

File: vagrant/provisioning/arkcase-all.yml [hunk 2]
```diff
@@ -114,10 +114,10 @@
       when: enable_microservices | default(false)
     - role: arkcase-ldap-authority
       tags: [core, upgrade, arkcase-ldap-authority, arkcase-microservices, marketplace]
-      when: enable_microservices | default(false) and enable_saml_microservice | default(false)
+      when: enable_microservices | default(false) and not (enable_saml_microservice | default(false))
     - role: arkcase-saml-authority
-      tags: [saml-authority]
-      when: enable_microservices | default(false)
+      tags: [arkcase-saml-authority]
+      when: enable_microservices | default(false) and enable_saml_microservice | default(false)
```
Comment (martinbozinovski) [inline vagrant/provisioning/arkcase-all.yml:120]: This role (arkcase-saml-authority) should have the same when clause as arkcase-ldap-authority or as it is defined now?

File: vagrant/provisioning/arkcase-ce-facts.yml [hunk 3]
```diff
@@ -630,4 +630,11 @@ file_management_s3_url: "http://acm-arkcase:9000"
 file_management_s3_access_key: "minioadmin"
 file_management_s3_secret_key: "minioadmin"
 file_management_s3_buckets:
-  - arkcase-portal
+  - config_name: arkcaseBucket
+    bucket_name: arkcase-portal
```
Comment (martinbozinovski) [inline vagrant/provisioning/arkcase-ce-facts.yml:634]: In all facts files, this bucket_name is without "-".

File: vagrant/provisioning/arkcase-ce-without-arkcase-facts.yml [hunk 4]
```diff
@@ -659,4 +659,11 @@ file_management_s3_url: "http://acm-arkcase:9000"
 file_management_s3_access_key: "minioadmin"
 file_management_s3_secret_key: "minioadmin"
 file_management_s3_buckets:
-  - arkcase-portal
+  - config_name: arkcaseBucket
+    bucket_name: arkcase-portal
```
Comment (martinbozinovski) [inline vagrant/provisioning/arkcase-ce-without-arkcase-facts.yml:663]: Same here, this should have "-" in front of the key.

File: vagrant/provisioning/arkcase-dev-facts.yml [hunk 5]
```diff
@@ -724,7 +724,15 @@ open_office_version: "4.1.11"
 enable_minio: true
 minio_console_port: 8001
 file_management_s3_url: "http://acm-arkcase:9000"
+file_management_s3_region: "us-east-1"
 file_management_s3_access_key: "minioadmin"
 file_management_s3_secret_key: "minioadmin"
 file_management_s3_buckets:
-  - arkcase-portal
+  - config_name: arkcaseBucket
+    bucket_name: arkcase-portal
```
Comment (martinbozinovski) [inline vagrant/provisioning/arkcase-dev-facts.yml:732]: Also here, need "-" in front of the bucket_name key.

File: vagrant/provisioning/roles/arkcase-email/templates/server.yml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+mail:
+  addresses:
+    from: "{{ email_user_from }}"
+  host: "{{ email_smtp_host }}"
+  port: {{ email_sender_port|int }}
```
Comment (martinbozinovski) [inline vagrant/provisioning/roles/arkcase-email/templates/server.yml]: Spaces around the pipe for readability

**Review by martinbozinovski — DISMISSED** [review]

File: vagrant/provisioning/arkcase-all.yml (hunk 2 above)
Comment (bojmil-armedia) [reply vagrant/provisioning/arkcase-all.yml:120]: It's not used now at all. But it should be either SAML od LDAP authentication. We might need both at some point, but currently it's one or the other.

File: vagrant/provisioning/arkcase-ce-facts.yml (hunk 3 above)
Comment (bojmil-armedia) [reply vagrant/provisioning/arkcase-ce-facts.yml:634]: This is one object in the list entry with two properties. Similar to ldap_groups fact.

File: vagrant/provisioning/roles/arkcase-email/templates/server.yml (hunk 6 above)
Comment (bojmil-armedia) [reply vagrant/provisioning/roles/arkcase-email/templates/server.yml]: OK

**Review by martinbozinovski — APPROVED** [review]

**Pull request merged**